.github/workflows/ci.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         run: |
           docker-compose exec backend poetry run pytest web_app/tests
 
+      - name: Run tests coverage
+        run: |
+          docker-compose exec backend poetry run pytest --cov=web_app/tests --cov-fail-under=90
```

Comment:
did you run it locally? 
outside of docker container

Reply:
Sent screenshot on Telegram.